consul/consul.go

```diff
@@ -60,6 +68,16 @@ func (r *ConsulAdapter) Register(service *bridge.Service) error {
 	registration.Tags = service.Tags
 	registration.Address = service.IP
 	registration.Check = r.buildCheck(service)
+	if r.servicePrefix != "" {
+		kv := r.client.KV()
+		for k, v := range service.Attrs {
+                        pair := &consulapi.KVPair{Key: r.servicePrefix + "/" + service.ID + "/" + k, Value: []byte(v)}
+			_, err := kv.Put(pair, nil)
+			if err != nil {
+				panic(err)
```

Comment:
what would happen here for the existing containers or next containers to appears ? I'm new to Go so I might misunderstood but it looks like the registrator would stop working. maybe returning `err` would be better ?